### ISSUE TITLE
Display recommendations when submitting user preferences

### ISSUE THREAD

**Issue by demir-delic** [chingu-voyages/v29-bears-team-07#29]
**__User Story__**
As a site visitor
I want my submitted preferences to result in recommendations
So I can find interesting new places to go on vacation.

**__Acceptance Criteria__**
- [x] When the "Find Vacation Spot" button on the landing page is clicked, user preferences are sent to the backend and recommendations are returned
- [x] Recommendations are displayed in a new page, with one card per recommendation
- [ ] Frontend does not assume number of recommendations
- [x] The new page has a "Search Again" button that navigates to the landing page
- [ ] In case of error, placeholder text is displayed
  * "Sorry, an error has occurred. Please try again later."
  * e.g. network error, no response from backend, no recommendations from backend, ...

**__Notes__**
Blocked by #28 
User preferences being persisted after a search is out of scope.
